app/src/main/java/com/soen390/conumap/ui/search_results/SearchResultsFragment.kt

```diff
@@ -145,7 +152,6 @@ class SearchResultsFragment : Fragment() {
 
         override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
-        viewModel = ViewModelProviders.of(this).get(SearchResultsViewModel::class.java)
```

Comment:
This is because this viewModel is not used yet and it is better for us to only use the SearchBarViewModel to communicate the data between fragments more easily